build-logic/convention/src/main/kotlin/com/google/samples/apps/nowinandroid/KotlinAndroid.kt

```diff
@@ -46,7 +47,7 @@ internal fun Project.configureKotlinAndroid(
 
         kotlinOptions {
             // Treat all Kotlin warnings as errors (disabled by default)
-            allWarningsAsErrors = properties["warningsAsErrors"] as? Boolean ?: false
+            allWarningsAsErrors = gradleLocalProperties(rootDir).getProperty("warningsAsErrors").toBoolean()
```

Comment:
Is this compatible with how we set this value in CI?

Reply:
Nope! It wasn't listening to either `local.properties` or `gradle.properties`. Fixed it to listen to both (tested by setting value in `local.properties`, in `~/.gradle/gradle.properties`, and removing in both). I had to set CI default to false, as our current project has a bunch of warnings